photon-lib/src/test/scala/com/linkedin/photon/ml/hyperparameter/search/GaussianProcessSearchTest.scala

```diff
@@ -204,14 +205,5 @@ object GaussianProcessSearchTest {
     def getEvaluationValue(result: TestModel): Double = result.evaluation
   }
 
-  val EVALUATOR: Evaluator = new Evaluator {
-
-    override protected[ml] val labelAndOffsetAndWeights: RDD[(Long, (Double, Double, Double))] =
-      mock(classOf[RDD[(Long, (Double, Double, Double))]])
-    override val evaluatorType: EvaluatorType = EvaluatorType.AUC
-    override protected[ml] def evaluateWithScoresAndLabelsAndWeights(
-        scoresAndLabelsAndWeights: RDD[(Long, (Double, Double, Double))]): Double = 0.0
-
-    override def betterThan(score1: Double, score2: Double): Boolean = score1 > score2
-  }
+  val IS_OPT_MAX = true
 }
```

Comment:
While you're editing this file, please add a newline at end of file

Reply:
Will do. Thanks.